packages/account/src/Sections/Security/SelfExclusion/_tests_/self-exclusion.spec.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { act } from 'react-dom/test-utils';
 import { fireEvent, render, screen, waitFor } from '@testing-library/react';
-import SelfExclusion from '../self-exclusion';
+import SelfExclusion from '..';
```

Comment:
is this change necessary? in other test files, i think we are following this pattern:
`import SelfExclusion from '../self-exclusion';`

Reply:
It was not reading the file for the some reason when I declared it like that

Reply:
can you double check this

Reply:
@sanjam-deriv Can you try wrapping the whole component in observer rather than having this:
export default observer(SelfExclusion);

Reply:
Changed this